third_party/mbedtls/iot_config_mbedtls.h

```diff
@@ -1,22 +1,47 @@
 /*
- * Copyright (C) 2019 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *  Copyright (C) 2006-2018, ARM Limited, All Rights Reserved
```

Comment:
Why has this license header changed?  Also, I wouldn't think that a file named `iot_` is part of mbedtls as stated on line 44.